mantidproject/mantid
# Remove custom dialog for CreateSampleShape

**Description of work.**

Removes the custom dialog for `CreateSampleShape`. The dialog is old and not used enough to warrant existence given the issues it causes.

**To test:**

Build MantidPlot and Workbench. Run the `CreateSampleShape` algorithm and see that the generic dialog with a simple text entry for the shape XML is offered.

Fixes #23620

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
